script.js

```diff
@@ -596,11 +679,45 @@ function handleRentStrikeLayer(geoJson) {
   return rentStrikeLayerMarkers;
 }
 
-// Ensures that map overlay pane layers are displayed in the correct Z-Order
-function fixZOrder(dataLayers) {
-  dataLayers.forEach(function (layerGroup) {
-    if (map.hasLayer(layerGroup)) {
-      layerGroup.bringToBack();
-    }
+function handleNationsLayer(geojson) {
+  // Scores are bound to range prop of each feature
+  const layerOptions = {
+    style: feature => {
+      const score = feature.properties.range
+      return {
+        // Get color from score
+        fillColor: scoreFillColors[score] || '#939393', // fallback color
```

Comment:
@river-honer this should use the same logic as the other layers for applying the fill color.

Reply:
I removed the above code blocks and switched to the boolean passed state for styling.